database/database.go

```diff
@@ -20,6 +20,58 @@ import (
 
 // TODO: prepare insert statements only once
 
+const createViews =
+// language=sql
+`
+CREATE VIEW allSwaps AS
```

Comment:
Is there `CREATE OR REPLACE VIEW` in SQLite?

Reply:
Unfortunately not, but it seems like you can hack around it with `DROP IF EXISTS